### PR TITLE
Update README as well as remove BOM encoding fixes

### PR DESCRIPTION
**Pull request by ShoGinn**
Just trying to provide a concise fix since the steps explained will cause errors if you do have credit cards.

I also updated the BOM byte hacks by specifying encoding.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,13 +37,13 @@ for you with a specific scenario, so others will know what works and what doesn'
 ## Preliminaries
 
 **In nYNAB:** Check if any of your category names contain a : or a `, as those are not valid characters in YNAB 4.
-Change these category names now. Also unhide all of your previously hidden categories, as
-the import will otherwise not work for them, and you'll have to re-categorise them manually. (Not a big deal because
+Change these category names now. Also un-hide all of your previously hidden categories, as
+the import will otherwise not work for them, and you'll have to re-categorize them manually. (Not a big deal because
```
Comment (rixx) [inline README.md]: Why this change?

File: README.md [hunk 2]
```diff
@@ -87,7 +98,7 @@ imports more likely.
 You'll want to import every file next, each under the appropriate account. Make sure to select Year/Month/Date as time
 format, as well as "Include transactions before account start date".
 
-Next, approve all transactions and recategorise if any did not receive a matching category on import. This shouldn't
+Next, approve all transactions and re-categorize if any did not receive a matching category on import. This shouldn't
```
Comment (rixx) [inline README.md]: Same here, why the change?

**Review by rixx — COMMENTED** [review]
Looking good on the encoding! – just curious about two of the spelling changes, what gives?

File: README.md (hunk 2 above)
Comment (ShoGinn) [reply README.md]: didn't even look at that! Used a spelling program on VScode ;) Guess it doesn't like the British spelling of things.

Categorize means to sort something into a particular group, to place something in a particular class. Categorize is the preferred North American spelling of this transitive verb, which is a verb that takes an object. Related words are categorizes, categorized, categorizing, categorization.

Categorise is the preferred British spelling, related words are categorises, categorised, categorising, categorisation. The North American spelling of categorize is also considered correct and is gaining acceptance around the world. Categorize and categorise are examples of a group of words that are spelled with a “z” in American English and with an “s” in British English.

Comment (rixx) [reply README.md]: I know that 'categorised' is preferred British spelling, that's why I'm baffled at having it removed.

**Review by rixx — APPROVED** [review]
